tests/generic_config_updater/test_dynamic_acl.py

```diff
@@ -122,9 +121,28 @@ class DHCP6OptClientLinkLayerAddr(_DHCP6OptGuessPayload):  # RFC6939
                    ShortField("lltype", 1),  # ethernet
                    _LLAddrField("clladdr", ETHER_ANY)]
 
-# Fixtures
 
+def increment_ipv4_addr(ipv4_addr, incr=1):
```

Comment:
Should we consider move the help function to utils/helpers, rather than copy it one more time?

Reply:
Maybe it's a better way, let me move